definitions/scenarios/upgrade_to_capsule_6_16.rb

```diff
@@ -38,6 +38,15 @@ class PreMigrations < Abstract
     end
 
     def compose
+      add_step(Procedures::Repositories::Setup.new(:version => '6.16'))
+      modules_to_enable = ["satellite-capsule:#{el_short_name}"]
+      add_step(Procedures::Packages::EnableModules.new(:module_names => modules_to_enable))
+      add_step(Procedures::Packages::Update.new(
+        :assumeyes => true,
+        :clean_cache => false,
```

Comment:
shouldn't it be `true` (default) here, and then `false` in Line 66?

Reply:
Errr, yes.